tests/modules/login_user/app/login_user_controller.test.ts

```diff
@@ -0,0 +1,73 @@
+import { it, expect, describe } from 'vitest';
+import { UserRepositoryMock } from '../../../../src/shared/infra/repositories/user_repository_mock';
+import { LoginUserUsecase } from '../../../../src/modules/login_user/app/login_user_usecase';
+import { LoginUserController } from '../../../../src/modules/login_user/app/login_user_controller';
+import { HttpRequest } from '../../../../src/shared/helpers/external_interfaces/http_models';
+
+describe('Assert Login User controller is correct at all', () => {
```

Comment:
Add tests for when the email and password are not valid, like wrong type or format, also should test with an correct email but wrong password